common/common.go

```diff
@@ -140,11 +140,9 @@ type Phase interface {
 	Season() Season
 	Type() PhaseType
 	Next() Phase
-	Prev() Phase
 	PostProcess(State) error
 	DefaultOrder(Province) Adjudicator
 	Options(Validator, Nation) (result Options)
-	Winner(Validator) *Nation
```

Comment:
Yes - this wasn't used either.  I think the function sits better as part of a variant, rather than as part of a phase, but the main reason I removed it was to avoid implementing it again.

Reply:
And if it was used we'd have another problem - it's bad to have the Winner check in two places, so removing it makes sense.